google_rest_api_generator/src/method_builder.rs

```diff
@@ -283,8 +284,8 @@ fn request_method<'a>(http_method: &str, params: impl Iterator<Item = &'a Param>
         fn _request(&self, path: &str) -> ::reqwest::RequestBuilder {
             let req = self.reqwest.request(#reqwest_method, path);
             #(let req = req.query(&[#query_params]);)*
-            // Hack until real oauth token support is implemented.
-            let req = req.bearer_auth(crate::auth_token());
+            let mut auth = self.auth.lock().unwrap();
+            let req = req.bearer_auth(auth.token::<_, String>(&[]).unwrap().access_token);
```

Comment:
See not above about creating crate::scope to replace crate::auth_token and passing the result of that into token here. That will continue to be a hack until the correct scope logic gets plumbed through, but will make the examples usable today.